easybuild/easyconfigs/i/IPython/IPython-3.0.0-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "PythonPackage"
+
+name = 'IPython'
+version = '3.0.0'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://archive.ipython.org/release/%(version)s/']
+
+python = 'Python'
+pyver = '2.7.9'
+pyshortver = '2.7'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('pysqlite', '2.6.3', versionsuffix),
+    ('PyZMQ', '14.5.0', '%s-zmq3' % versionsuffix),
+    ('requests', '2.6.0', versionsuffix)
+]
+
+# override extensions sanity check, default filter that imports a Python module doesn't work here
+exts_filter = ('ipython -h', "")
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%s/site-packages/IPython' % pyshortver],
+}
+
+sanity_check_commands = [('iptest','')]
```

Comment:
hmm, hold on... this may not work since (the right) `iptest` will not be included in `$PATH` yet, since `runtest` is used right after the `build_step` (before actually having installing IPython, or before having a module file for it)